conductor/src/main.rs

```diff
@@ -54,6 +54,27 @@ fn main() {
                     "Successfully loaded {} instance configurations",
                     conductor.instances().len()
                 );
+
+                // Checking if there is a dpki instance
+                if conductor.using_dpki() {
+                    println!("Starting dpki instances...");
+                    conductor
+                        .start_dpki_happ_instance()
+                        .expect("Could not start dpki happ instance!");
+
+                    conductor
+                        .dpki_bootstrap()
+                        .expect("Could not bootstrap dpki!");
+
+                    // Need to stop the dpki instance for now.
```

Comment:
Instead of stopping the DPKI instance, how about passing in a boolean "dpki_instance_already_started" that skips starting it?  Would that work?

Reply:
I would suggest we change `Conductor::start_all_instances()` to ignore *already-running* errors and then leave the DPKI instance running here...